src/Node/Express/Handler.purs

```diff
@@ -37,283 +39,271 @@ import Node.Express.Internal.Response
 import Node.Express.Internal.Request
 import Node.Express.Internal.QueryString
 
---| Monad responsible for handling single request.
-data HandlerM a = HandlerM (Request -> Response -> ExpressM Unit -> ExpressM a)
-type Handler = HandlerM Unit
+-- | Monad responsible for handling single request.
+data HandlerM e a = HandlerM (Request -> Response -> Eff e Unit -> Aff e a)
```

Comment:
Shouldn't here be not just `e`, but `(express :: EXPRESS | e)` on the right side?